From fc9db7adcfd273dfde7abb3c541d4c77378cbe2b Mon Sep 17 00:00:00 2001
From: Matthias Brugger <mbrugger@suse.com>
Date: Mon, 27 Jun 2016 18:24:33 +0200
Subject: arm64: acpi: Add acpi_numa_arch_fixup() stub
Patch-mainline: No, it is reworked to not touch generic code
References: fate#319981

This patch adds acpi_numa_arch_fixup() stub to arm64.

It is loosley based on:
312521d05435 ("ACPI / NUMA: Move acpi_numa_arch_fixup() to ia64 only")

Signed-off-by: Matthias Brugger <mbrugger@suse.com>
---
 arch/arm64/kernel/acpi.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/kernel/acpi.c b/arch/arm64/kernel/acpi.c
index d1ce8e2..bbbbc8f 100644
--- a/arch/arm64/kernel/acpi.c
+++ b/arch/arm64/kernel/acpi.c
@@ -44,6 +44,8 @@ EXPORT_SYMBOL(acpi_pci_disabled);
 static bool param_acpi_off __initdata;
 static bool param_acpi_force __initdata;
 
+void __init acpi_numa_arch_fixup(void) {}
+
 static int __init parse_acpi(char *arg)
 {
 	if (!arg)
-- 
2.8.4

